doc/nrf/libraries/others/network_core_monitor.rst

```diff
@@ -68,38 +74,28 @@ Usage
 
 To enable the Network core monitor library, set the :kconfig:option:`CONFIG_NET_CORE_MONITOR` Kconfig option.
 
-On the application core, periodically call the :c:func:`ncm_net_status_check` function to monitor the state of the network core.
+The :c:func:`ncm_net_core_event_handler` function can be used to notify the application about the status of the network core.
+To define the user action for the event, you must override the ``__weak`` function definition of the :c:func:`ncm_net_core_event_handler`.
```

Comment:
@peknis  ?
```suggestion
To define the user action for the event, you must override the ``__weak`` function definition of the :c:func:`ncm_net_core_event_handler` function.
```

Reply:
Maybe this is a bit of an overkill, now that we already talk about a function definition. Maybe you can leave it without the definite article "the" in front of the function name and the word "function" at the end.

Reply:
Thanks for the clarification.